Controllers/ExtensionMappers/ExtensionMappers.cs

```diff
@@ -0,0 +1,43 @@
+﻿using EcommerceStore.Controllers.InputModels;
+using EcommerceStore.Controllers.ViewModels;
+using EcommerceStore.Data.Entities;
+using System;
+
+namespace EcommerceStore.Controllers.ExtensionMappers
+{
+    public static class ExtensionMappers
+    {
+        public static Brand MapToBrand(this BrandInputModel inputModel)
+        {
+            if (inputModel != null)
```

Comment:
no need to validate input model here, as it should have been validated in controller